bitbots_ceiling_cam/config/camera_calibration_ceiling_cam.yaml

```diff
@@ -0,0 +1,20 @@
+image_width: 2048
+image_height: 1536
+camera_name: deckenkamera
```

Comment:
change camera name to ceiling_cam?

Reply:
I tried but it fails if its not called "deckenkamera". I seems that this name is configured inside the camera itself. Maybe it is possible to reconfigure with the pylon app?